recipe/meta.yaml

```diff
@@ -41,12 +40,12 @@ outputs:
       requires:
         - pip
 
-  - name: py-rattler
+  - name: py_rattler
     build:
       noarch: generic
     requirements:
```

Comment:
If you want this to have an exact pin, you also need to add the pin to `host` to workaround the solvability issue.